lib/galaxy_test/api/test_tool_data.py

```diff
@@ -48,6 +57,14 @@ def test_download_field_file(self):
         content = show_field_response.text
         assert content == "This is data 1.", content
 
+    def test_download_field_file_anon_raises_404(self):
+        show_field_response = self._get("tool_data/twobit/fields/data1/files/entry.txt")
+        self._assert_status_code_is(show_field_response, 404)
+
+    def test_download_field_file_anon_raises_403(self):
```

Comment:
You shouldn't be anon here. I think the special handling (admin=True) in test_download_field_file